src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+fieldset {
```

Comment:
checklist p.6 Don't style tags, use class selector instead

Reply:
i have just removed

Reply:
not fixed